docs/README.md

```diff
@@ -0,0 +1,51 @@
+# Documentation
+
+<!-- toc is generated with GitHub Actions do not remove toc markers -->
+
+<!-- toc -->
+
+- [Usage (`./usage.md`)](#usage-usagemd)
+- [Advanced (`./advanced.md`)](#advanced-advancedmd)
+- [Integration (`./integration.md`)](#integration-integrationmd)
+- [Development (`./development.md`)](#development-developmentmd)
+- [Input processing (`./input_processing.md`)](#input-processing-input_processingmd)
+- [Generators (`./generators.md`)](#generators-generatorsmd)
+- [Presets (`./presets.md`)](#presets-presetsmd)
+- [Interpretation of JSON Schema draft 7 (`./interpretation_of_JSON_Schema_draft_7.md`)](#interpretation-of-json-schema-draft-7-interpretation_of_json_schema_draft_7md)
+- [Languages](#languages)
+
+<!-- tocstop -->
+
+This document gives the overview of all the available documentation for Modelina.
+
+### [Usage (`./usage.md`)](./usage.md)
```

Comment:
I am not sure I like this format.. But I feel like the link to the file should be visible somehow. What do you think? 🤔

Reply:
<img width="1304" alt="Screen Shot 2021-09-02 at 10 16 51 AM" src="https://user-images.githubusercontent.com/19964402/131888343-84d10c53-e7c1-4d35-acd7-5d0acaad8430.png">

How about something like this? You still link to the markdown file from the title and the link is visible, but you do not include the file name in the title.

Reply:
I guess we have to stick with that yea, it looks better 😄